### DISCUSSION TITLE
How long will Dropwizard v1 continue to be supported?

### DISCUSSION THREAD

**Discussion by mikeholler** [dropwizard/dropwizard#4758] [General]
I opened another issue (#3335) that I believe is blocking for me to upgrade to Dropwizard 2. After encountering this issue and looking around, I wanted to see if 1.3.X is continuing to receive security updates, etc.

Currently, the migration to v2 seems to be a substantial effort (see #3335), one that I don't have time for at the moment. However, it is important to me that we complete the update before security fixes stop for the 1.3 branch.

In looking at the releases history, it looks like 1.3.27 was released after 2.0 came out, which was great to see. However, I am still curious whether there are clear plans for continuing to provide updates and backports for v1. Is this documented some place that I am missing?

BTW aside from creating a couple of issues recently, I've been really enjoying working with this project. Thank you for all the hard work!

**Comment by mattnelson** [comment]
> Is this documented some place that I am missing?

I looked here to see the security policy, which calls out which versions are supported.
https://github.com/dropwizard/dropwizard/security/policy

**Comment by mikeholler** [comment]
Thank you, this is great. I can see that it is currently supported, but I am wondering if there is any plan for sunsetting v1.3 in the near- or mid- future. For example, can we reasonably expect that 1.3 will be supported through the end of the year? Longer? Shorter? Basically, I'd like to know as much as there is to know about the projected end of support (or whatever the maintainers know about it) so we can prepare and aren't surprised by it and can plan our updates accordingly :slightly_smiling_face:

**Comment by joschi** [comment]
@mikeholler Thanks for this question and sorry that it took us so long to come back to you.

Dropwizard 1.3.x will be supported until December 31, 2020.

Please see https://groups.google.com/g/dropwizard-user/c/6gpDMuSb4_Y/m/poqbA4T9BQAJ for the official announcement on the dropwizard-user mailing list.
